### ISSUE TITLE
some inconsistency with the paper

### ISSUE THREAD

**Issue by Okazaki86** [hzhupku/RegionContrast#5]
1. in the paper it calculates class centers within a single image, but implements as within a batch, which just return a matrix of (class, feature dim) for method dec_deeplabv3_contrast.contrast_region.
2. the variable _l_pos_ in dec_deeplabv3_contrast.forward when calculating contrast loss should be a similarity score of 1 dim, but it shows still 256 dim in the code.

**Comment by liangfengyouxingii** [comment]
@Okazaki86  hello, I also notice this problem, do you solve it?

**Comment by liangfengyouxingii** [comment]
may l_pos = query.unsqueeze(0) @ eval("self.queue" + str(cls_ind)).clone().detach()

**Comment by Okazaki86** [comment]
> @Okazaki86 hello, I also notice this problem, do you solve it?

我按论文里的方法进行了改正
